services/src/main/java/com/chatbot/services/AsyncServices/HangoutsAsyncService.java

```diff
@@ -0,0 +1,144 @@
+package com.chatbot.services.AsyncServices;
+
+import java.io.IOException;
+import java.util.List;
+
+import com.chatbot.services.ChatServiceConstants;
+import com.chatbot.services.IDMapping;
+import com.chatbot.services.DialogflowServices.DialogflowConversation;
+import com.chatbot.services.MessageSenders.HangoutsMessageSender;
+import com.chatbot.services.MessageSenders.HangoutsMessageSender.HANGOUTS_MESSAGE_TYPE;
+import com.chatbot.services.protobuf.ChatServiceRequestOuterClass.ChatServiceRequest;
+import com.chatbot.services.protobuf.ChatServiceRequestOuterClass.ChatServiceRequest.ChatClient;
+import com.chatbot.services.protobuf.TriggerEventNotificationOuterClass.TriggerEventNotification;
+import com.google.cloud.dialogflow.v2.QueryResult;
+import com.google.protobuf.Struct;
+import com.google.protobuf.Value;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Component;
+
+@Component
+public class HangoutsAsyncService extends AsyncService {
+
+  @Autowired
+  private HangoutsMessageSender hangoutsMessageSender;
+  @Autowired
+  private IDMapping iDMapping;
+  @Autowired
+  private DialogflowConversation dialogflowConversation;
+
+  // handler for a request from the chat client
+  @Override
```

Comment:
If you are overriding a method/function, no need to explicitly add comment here, the comments from the parent class hold true for this

Reply:
updated